moveit_core/robot_model/src/joint_model_group.cpp

```diff
@@ -326,9 +326,10 @@ void JointModelGroup::getVariableRandomPositionsNearBy(random_numbers::RandomNum
 {
   assert(active_joint_bounds.size() == active_joint_model_vector_.size());
   for (std::size_t i = 0; i < active_joint_model_vector_.size(); ++i)
-    active_joint_model_vector_[i]->getVariableRandomPositionsNearBy(
-        rng, values + active_joint_model_start_index_[i], *active_joint_bounds[i],
-        near + active_joint_model_start_index_[i], distance);
+    active_joint_model_vector_[i]->getVariableRandomPositionsNearBy(rng, values + active_joint_model_start_index_[i],
+                                                                    *active_joint_bounds[i],
+                                                                    near + active_joint_model_start_index_[i],
+                                                                    distance);
```

Comment:
i prefer the old version - this change is less readable IMO and adds another line.